installer/tue-install-impl.bash

```diff
@@ -655,13 +675,14 @@ function tue-install-ppa
     then
         tue-install-error "Invalid tue-install-ppa call: needs ppa as argument."
     fi
+    local ppa="$*"
 
-    if [[ ! $1 == ppa:* ]]
+    if [[ $ppa != ppa:* && $ppa != deb* ]]
```

Comment:
This needs to be added in the docs that `deb` sources can now be installed using `tue-install-ppa`